moveit_planners/stomp/examples/stomp_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,19 @@
+<launch>
```

Comment:
move this into the setup assistant template

Reply:
added !